File: DTOs/Sources/MeetingServiceDTOs/GetLocationDTO.swift
import Foundation

public struct GetLocationDTO: Codable {
    public var id: UUID
    public var name: String
    public var street: String
    public var number: String
    public var letter: String
    public var postalCode: String?
    public var place: String?
    
    public init(id: UUID, name: String, street: String, number: String, letter: String, postalCode: String? = nil, place: String? = nil) {
        self.id = id
        self.name = name
        self.street = street
        self.number = number
        self.letter = letter
        self.postalCode = postalCode
        self.place = place
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/MeetingServiceDTOs/GetLocationDTO.kt
package net.ipv64.kivop.dtos.MeetingServiceDTOs

import java.util.UUID

data class GetLocationDTO(
    var id: UUID,
    var name: String,
    var street: String,
    var number: String,
    var letter: String,
    var postalCode: String?,
    var place: String?,
)
